### DISCUSSION TITLE
离线安装无法运行

### DISCUSSION THREAD

**Discussion by pangjian** [he3-app/he3-feedback#131] [Q&A]
离线安装（用安装包安完以后不联网），无法运行，一直白屏

**Comment by lyzhang1999** [comment]
你好，目前 He3 客户端是需要联网才能使用的，离线使用的问题我们正在进一步评估。

issues：https://github.com/he3-app/he3-feedback/issues/132

**Reply by Cang-Cat** [reply]
1.2.9还支持离线的，结果安装了新的反而不能使用了。

**Reply by lovitus** [reply]
> 1.2.9还支持离线的，结果安装了新的反而不能使用了。



> 1.2.9还支持离线的，结果安装了新的反而不能使用了。

谢谢回答...   付费,授权都能接受,  离线不能用就太烦了
